docs/finops/README.md

```diff
@@ -0,0 +1,46 @@
+---
+title: FinOps
+description: DevOps-like role in a collaborative approach to optimize cloud costs, ensuring financial accountability, efficiency, and resource management.
+sidebar_position: 9
+---
+
+# FinOps
+
+Understand financial principles relevant to cloud computing, like budgeting, forecasting, cost allocation, and billing.
+
+**Tools offered by cloud providers**:
```

Comment:
There should be an H2 header (starts with `##`) for AWS cloud as all content here is about AWS.

```
## Cloud - AWS
```